scripts/build_android_libtorch_and_pytorch_android.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+set -eux
+
+PYTORCH_DIR="$(cd $(dirname $0)/..; pwd -P)"
+
+PYTORCH_ANDROID_DIR=$PYTORCH_DIR/android
+WORK_DIR=$PYTORCH_DIR
+
+echo "PYTORCH_DIR:$PYTORCH_DIR"
+echo "WORK_DIR:$WORK_DIR"
+
+echo "ANDROID_HOME:$ANDROID_HOME"
```

Comment:
Where is ANDROID_HOME used? Is it required by gradle or what?

Reply:
Yes, gradle needs either ANDROID_HOME or sdk.dir in gradle properties - path to the android_sdk to use